### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by LyriCaGitHubber**
I created the header component that shows the **sun icon** and the **header text**.
Additionally i create the **css module** for this component.

### CODE REVIEW

**Review by christophschmitz — APPROVED** [review]
Looks just like the design! Very good

File: src/app/assets/SunIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function SunIcon(): JSX.Element {
+  return (
+    <svg
+      width="42"
+      height="42"
+      viewBox="0 0 42 42"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (marwinburesch) [inline src/app/assets/SunIcon.tsx:10]: You don't want to pass props?

**Review by marwinburesch — COMMENTED** [review]
isNice = true

File: src/app/assets/SunIcon.tsx (hunk 1 above)
Comment (LyriCaGitHubber) [reply src/app/assets/SunIcon.tsx:10]: I thought about it but i think i don´t need properties in this component. The header is static. Or have you any ideas what i could pass? :)

**Pull request merged**